CHANGELOG.md

```diff
@@ -27,11 +28,43 @@ Bug Fixes / Other Changes:
   * Azure: remove `location` requirement from Ark config (#344, @skriss)
   * Add documentation/examples for storing backups in IBM Cloud Object Storage (#321, @roytman)
   * Reduce verbosity of hooks logging (#362, @skriss)
-  * AWS: Add minimal IAM policy to documentation (#363, @hopkinsth)
+  * AWS: Add minimal IAM policy to documentation (#363 #419, @hopkinsth)
   * Don't restore events (#374, @sanketjpatel)
   * Azure: reduce API polling interval from 60s to 5s (#359, @skriss)
   * Switch from hostPath to emptyDir volume type for minio example (#386, @containscafeine)
   * Add limit ranges as a prioritized resource for restores (#392, @containscafeine)
+  * AWS: Add documentation on using Ark with kube2iam (#402, @domderen)
+  * Azure: add node selector so Ark pod is scheduled on a linux node (#415, @ffd2subroutine)
+  * Error from the Ark CLI if attempting to get logs for a non-existent restore (#391, @containscafeine)
+  * GCP: Add minimal IAM policy to documentation (#429, @skriss @jody-frankowski)
+
+##### Upgrading from v0.7.1:
+  Ark v0.7.1 moved the Ark server deployment into a separate namespace, `heptio-ark-server`. As of v0.8.0 we've
+  returned to a single namespace, `heptio-ark`, for all Ark-related resources. If you're currently running v0.7.1,
+  here are the steps you can take to upgrade:
+
+1. Execute the steps from the **Credentials and configuration** section for your cloud:
+    * [AWS](https://github.com/heptio/ark/blob/v0.8.0/docs/aws-config.md#credentials-and-configuration)
+    * [Azure](https://github.com/heptio/ark/blob/v0.8.0/docs/azure-config.md#credentials-and-configuration)
+    * [GCP](https://github.com/heptio/ark/blob/v0.8.0/docs/gcp-config.md#credentials-and-configuration)
+
+    When you get to the secret creation step, if you don't have your `credentials-ark` file handy, 
+    you can copy the existing secret from your `heptio-ark-server` namespace into the `heptio-ark` namespace:
+    ```bash
+    kubectl get secret/cloud-credentials -n heptio-ark-server --export -o json | \
+      jq '.metadata.namespace="heptio-ark"' | \
+      kubectl apply -f -
+    ```
+
+2. You can now safely delete the `heptio-ark-server` namespace:
+    ```bash
+    kubectl delete namespace heptio-ark-server
+    ```
+
+3. Execute the commands from the **Start the server** section for your cloud:
+    * [AWS](https://github.com/heptio/ark/blob/v0.8.0/docs/aws-config.md#start-the-server)
+    * [Azure](https://github.com/heptio/ark/blob/v0.8.0/docs/azure-config.md#start-the-server)
+    * [GCP](https://github.com/heptio/ark/blob/v0.8.0/docs/gcp-config.md#start-the-server)
 
```

Comment:
In the preceding lines, you're linking explicitly to the markdown files in the GH repository. I can construct the URL for the equivalent links to the docs site, which might be better. We should also probably get these steps into the docs very soon -- maybe next week after other releases are out the door? Easy enough to rebuild for the same version.

Reply:
:+1: to using the docs site URL vs repo files.

Reply:
Good point, will do.